BridgeAppSDK/SBAScheduledActivityManager.swift

```diff
@@ -1075,6 +1088,25 @@ open class SBABaseScheduledActivityManager: NSObject, ORKTaskViewControllerDeleg
         }
     }
     
+    /**
+     If the task is available right away, the callback will be invoked
+     Otherwise, we will save the callback to be executed later when today's task is available
+    */
+    public func notifyWhenTaskIsAvailable(taskId: String, callback: @escaping () -> Void) {
```

Comment:
If not intended to be overridable, mark it as final.  

Additionally, please don't call this from the private load and set the callback and id again. And what happens if the developer sets this up to listen for more than one task id? Then this will get clobbered and end up with the closure not getting called at all.